docs/source/1.0/spec/core/behavior-traits.rst

```diff
@@ -655,3 +655,95 @@ See
             input: PutSomethingInput,
             output: PutSomethingOutput
         }
+
+
+.. _httpChecksum-trait:
+
+----------------------
+``httpChecksum`` trait
+----------------------
+
+Summary
+    Indicates that an operation's HTTP Request or Response supports checksum
+    validation. At least one of request or response checksum properties must
+    be specified within the trait.
+Trait selector
+    ``operation``
+Value type
+    ``structure``
+
+The `httpChecksum` trait is a structure that contains the following members:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 10 10 80
+
+    * - Property
+      - Type
+      - Description
+    * - Request
```

Comment:
Can you make these lowercase?

Reply:
updated `Request` -> `request`, and `Response` -> `response`.